app/views/admin/participants/_nav.html.erb

```diff
@@ -7,27 +7,16 @@
     Training
   <% end %>
 
-  <%= component.with_nav_item(path: admin_participant_statuses_path(@participant_profile)) do %>
-    Statuses
-  <% end %>
-
-  <%= component.with_nav_item(path: admin_participant_induction_records_path(@participant_profile)) do %>
-    Induction records
-  <% end %>
-
   <%= component.with_nav_item(path: admin_participant_declaration_history_path(@participant_profile)) do %>
-    Declaration history
+    Declarations
   <% end %>
 
   <%= component.with_nav_item(path: admin_participant_validation_data_path(@participant_profile)) do %>
     Validation data
```

Comment:
The nav  'Validation data' item is not on the prototype. Does it stay, or is it removed?